vllm/model_executor/model_loader/loader.py

```diff
@@ -1504,4 +1525,7 @@ def get_model_loader(load_config: LoadConfig) -> BaseModelLoader:
     if load_config.load_format == LoadFormat.RUNAI_STREAMER:
         return RunaiModelStreamerLoader(load_config)
 
+    if load_config.load_format == LoadFormat.RUNAI_STREAMER_SHARDED:
+        return ShardedStateLoader(load_config, True)
```

Comment:
```suggestion
        return ShardedStateLoader(load_config, runai_model_streamer=True)
```

Make the boolean argument explicit